service/controller/key/key.go

```diff
@@ -104,6 +105,13 @@ const (
 	LegacyLabelCluster = "cluster"
 )
 
+const (
+	// TODO: Change to 1.16.8
+	kubectlVersion               = "9ccdc9dc55a01b1fde2aea73901d0a699909c9cd" // 1.15.5
+	KubernetesNetworkSetupDocker = "9ccdc9dc55a01b1fde2aea73901d0a699909c9cd"
```

Comment:
```suggestion
	KubernetesNetworkSetupDocker = "68e90113331feca3b9ffe6a75a601b381ba8c1f7"
```

Reply:
Was already on the way :)

Reply:
Nice--didn't meant to step on your toes. It just caught my eye that the tags were identical between this and `kubectl` above.

Reply:
No worries, thanks for the help. Trying with the new version now

Reply:
All three images are at `68e90113331feca3b9ffe6a75a601b381ba8c1f7` now. That's probably not intended.

Reply:
Cluster is up :) those were pinned to the same version originally so I aligned them again

Reply:
Sounds good.